src/cli/cli.cpp

```diff
@@ -705,6 +708,243 @@ template <> otError Interpreter::Process<Cmd("br")>(Arg aArgs[])
 }
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
 
+#if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE
+template <> otError Interpreter::Process<Cmd("nat64")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    if (aArgs[0].IsEmpty())
+    {
+        ExitNow(error = OT_ERROR_INVALID_COMMAND);
+    }
+    /**
+     * @cli nat64 configuredcidr
+     * @code
+     * nat64 configuredcidr
+     * 192.168.64.0/24
+     * Done
+     * @endcode
+     * @par
+     * Gets the NAT64 IPv4 CIDR used by NAT64 translator.
+     * @par
+     * Available when `OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE` is enabled.
+     * @par api_copy
+     * #otNat64GetConfiguredCidr
+     *
+     */
+    else if (aArgs[0] == "configuredcidr")
+    {
+        otIp4Cidr cidr;
+        char      cidrString[OT_IP4_CIDR_STRING_SIZE];
+
+        SuccessOrExit(error = otNat64GetConfiguredCidr(GetInstancePtr(), &cidr));
+        otIp4CidrToString(&cidr, cidrString, sizeof(cidrString));
+        OutputLine("%s", cidrString);
+    }
+    /**
+     * @cli nat64 configuredprefix
+     * @code
+     * nat64 configuredprefix
+     * fd56:eda7:17ca:2:0:0::/96
+     * Done
+     * @endcode
+     * @par
+     * Gets the NAT64 IPv6 prefix used by NAT64 translator.
+     * @par
+     * Available when `OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE` is enabled.
+     * @par api_copy
+     * #otNat64GetConfiguredPrefix
+     *
+     */
+    else if (aArgs[0] == "configuredprefix")
+    {
+        otIp6Prefix prefix;
+
+        SuccessOrExit(error = otNat64GetConfiguredPrefix(GetInstancePtr(), &prefix));
+        OutputIp6PrefixLine(prefix);
+    }
+    /**
+     * @cli nat64 mappings
+     * @code
+     * nat64 mappings
+     * |          | Address                   |        | 4 to 6       | 6 to 4       |
+     * +----------+---------------------------+--------+--------------+--------------+
+     * | ID       | IPv6       | IPv4         | Expiry | Pkts | Bytes | Pkts | Bytes |
+     * +----------+------------+--------------+--------+------+-------+------+-------+
+     * | 00021cb9 | fdc7::df79 | 192.168.64.2 |  7196s |    6 |   456 |   11 |  1928 |
+     * |          |                                TCP |    0 |     0 |    0 |     0 |
+     * |          |                                UDP |    1 |   136 |   16 |  1608 |
+     * |          |                               ICMP |    5 |   320 |    5 |   320 |
+     * @endcode
+     * @par
+     * Gets the current address mappings of NAT64 translator.
+     * @par
+     * Available when `OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE` is enabled.
+     * @par api_copy
+     * #otNat64GetNextAddressMapping
+     *
+     */
+    else if (aArgs[0] == "mappings")
+    {
+        otNat64AddressMappingIterator iterator = OT_NETWORK_DATA_ITERATOR_INIT;
+        otNat64AddressMapping         mapping;
+        uint32_t                      currentTime = otPlatAlarmMilliGetNow();
```

Comment:
Suggest against using `otPlatAlaramMilliGetNow()` here.  

Looks like it is used to convert `mExpiry` to remaining time.
Instead may better that in `otNat64AddressMapping` we return the remaining time directly. 

User of the public OT API should be able to use the API without knowing about platform APIs and may not have any notion of `AlarmMilliGetNow()`.